src/services/item/plugins/geolocation/repository.ts

```diff
@@ -71,7 +71,8 @@ export class ItemGeolocationRepository {
 
     const geoloc = this.repository
       .createQueryBuilder('ig')
-      .leftJoinAndSelect('ig.item', 'item')
+      // inner join to filter out recycled items
+      .innerJoinAndSelect('ig.item', 'item')
```

Comment:
Ah yes ... this is the real pain of using the soft-delete feature ... I recall having to hunt down usage like these with Morgan ... Not fun.